packages/eas-cli/schema/metadata-0.json

```diff
@@ -464,62 +464,43 @@
       },
       "AppleRelease": {
         "type": "object",
+        "description": "Configure how the app is released in the app store.",
         "additionalProperties": false,
-        "defaultSnippets": [
-          {
-            "body": {
-              "automaticRelease": true,
-              "usesThirdPartyContent": false,
-              "usesNonExemptEncryption": false
-            }
-          }
-        ],
         "properties": {
-          "autoReleaseDate": {
-            "description": "ISO Date to release the app automatically. Leave undefined to manually release the app to the public.",
-            "defaultSnippets": [
+          "automaticRelease": {
+            "description": "If and how the app should automatically be released when approved by App Store review.",
+            "markdownDescription": "If and how the app should automatically be released when approved by App Store review.\n- `false` - Manually release the app after store approval. (default behavior)\n- `true` - Automatically release after store approval.\n- `Date` - Automatically schedule release on this date, after store approval.\n\nApple does not guarantee that your app is available at the chosen scheduled release date.",
```

Comment:
This is technically not a `Date` object in JSON, maybe need to update to be an example string?